bang/models.py

```diff
@@ -1166,11 +1166,15 @@ class Song(MagiModel):
 
     owner = models.ForeignKey(User, related_name='added_songs')
     _original_image = models.ImageField(null=True, upload_to=uploadTiny('s'))
-    image = models.ImageField('Album cover', upload_to=uploadItem('s'))
+    image = models.ImageField(_('Album cover'), upload_to=uploadItem('s'))
```

Comment:
Why is this translatable? It's not visible anywhere for users.

Reply:
I just thought it should be translatable in case any staff members were using the site in another language. I know it's unlikely, but I thought it'd just be better safe then sorry. If you want me to though, I can quickly un-translate it.

Reply:
All terms that are staff-only are currently not translated because we already have too much work for translators.